client/static/css/style.css

```diff
@@ -0,0 +1,156 @@
+/* uchen-regular - latin */
+@font-face {
+	font-family: 'Uchen';
+	font-style: normal;
+	font-weight: 400;
+	src: local(''), url('../fonts/uchen-v1-latin-regular.woff2') format('woff2'),
+		/* Chrome 26+, Opera 23+, Firefox 39+ */ url('../fonts/uchen-v1-latin-regular.woff')
+			format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+:root {
+	--background-color: #f8e2cf;
+	--input-color: #fdf6f0;
+	--off-white: #faf8f6;
+	--text-color: #381806;
+	--placeholder-color: #dd8451;
+}
+
+* {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body {
+	font-family: 'Uchen', serif;
+	min-height: 100vh;
+	background-color: var(--background-color);
+	color: var(--text-color);
+	padding-top: 50px;
+}
+
+h1 {
+	width: fit-content;
+	margin: 0 100px 20px;
+	font-size: 50px;
+	transition: 0.3s;
+}
+
+h1:hover {
+	color: var(--placeholder-color);
+}
+
+h1 a {
+	color: inherit;
+	text-decoration: none;
+}
+
+form,
+article {
+	margin-left: 200px;
+}
+
+form {
+	width: 500px;
+}
+
+label,
+input {
+	display: block;
+	font-family: 'Uchen', serif;
+}
+
+label {
+	text-transform: capitalize;
+	font-size: 22px;
+}
+
+input {
+	border: none;
+	padding: 0 10px;
+	border-radius: 10px;
+	font-size: 20px;
+	background-color: var(--input-color);
+	width: 100%;
+	transition: 0.2s;
+}
+
+input::placeholder {
+	text-transform: capitalize;
+	color: var(--placeholder-color);
+}
+
+input:focus,
+input:focus-visible {
+	background-color: var(--off-white);
+	outline: none;
+	box-shadow: none;
+}
+
+input[type='submit'] {
+	margin: 20px 0;
+	width: auto;
+	cursor: pointer;
+}
+
+input[type='submit']:hover {
+	background-color: var(--off-white);
+	color: var(--placeholder-color);
+}
+
+article {
+	width: 500px;
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+}
+
+h2 {
+	padding-bottom: 10px;
+	margin-bottom: 10px;
+	border-bottom: 1px solid var(--placeholder-color);
+	flex-basis: 100%;
+	font-size: 32px;
+}
+
+p:nth-child(2),
+p:nth-child(3) {
+	flex-basis: 50%;
+	display: inline;
+}
+
+p:nth-child(3) {
+	text-align: right;
+}
+p:nth-child(4) {
+	width: 100%;
+	padding-top: 15px;
+	margin-top: 10px;
+	border-top: 1px solid var(--placeholder-color);
+}
+
```

Comment:
Nice that you have decided to include some media queries in here. Responsiveness could be slightly better, seemed to cut off some content on smaller screens.